test/pylib/manager_client.py

```diff
@@ -304,6 +306,13 @@ async def server_change_ip(self, server_id: ServerNum) -> IPAddress:
                                          response_type="json")
         return IPAddress(ret["ip_addr"])
 
+    async def server_change_rpc_address(self, server_id: ServerNum) -> IPAddress:
+        """Change server RPC IP address. Applicable only to a stopped server"""
+        logger.debug("ManagerClient changing RPC IP for server %s", server_id)
```

Comment:
It would be more informative if we print this message after we get the new address and include it the message.

Reply:
changed